gremlin-go/driver/driverRemoteConnection.go

```diff
@@ -181,10 +181,28 @@ func (driver *DriverRemoteConnection) CreateSession(sessionId ...string) (*Drive
 	drc, err := NewDriverRemoteConnection(driver.client.url, func(settings *DriverRemoteConnectionSettings) {
 		settings.TraversalSource = driver.client.traversalSource
 		if len(sessionId) == 1 {
-			settings.Session = sessionId[0]
+			settings.session = sessionId[0]
 		} else {
-			settings.Session = uuid.New().String()
+			settings.session = uuid.New().String()
 		}
+
+		settings.TraversalSource = driver.client.traversalSource
+		settings.TransporterType = driver.client.transporterType
+
+		settings.Logger = driver.client.logHandler.logger
+		settings.LogVerbosity = driver.client.logHandler.verbosity
+		//settings.Language = driver.client.logHandler
```

Comment:
Should we keep the same logger and verbosity as parent DRC? Language isnt stored in DRC, default to English?